daemon/daemon.go

```diff
@@ -905,6 +899,43 @@ func (d *Daemon) init() error {
 	return nil
 }
 
+// syncLXCMap adds local host enties to bpf lxcmap, as well as
+// ipcache, if needed, and also notifies the daemon and network policy
+// hosts cache if changes were made.
+func (d *Daemon) syncLXCMap() error {
+	// TODO: Update addresses first, in case node addressing has changed.
+	// TODO: Once these start changing on runtime, figure out the locking strategy.
```

Comment:
What specifically are we deferring figuring out here? Locking of `lxcmap`? of `IPIdentityCache`?